src/net-utils.ts

```diff
@@ -72,6 +73,9 @@ export async function openBrowserWhenReady(
     // No need to wait for Shiny app to start or open the browser
     return;
   }
+  if (timeout === 10000) {
+    timeout = configShinyTimeout();
```

Comment:
You could instead make `configShinyTimeout()` the default in the function signature, i.e.

```
timeout: number = configShinyTimeout()
```